theme/ckeditor5-ui/globals/_reset.css

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+@custom-selector :--ck-reset-all
+	.ck-reset_all,
+	.ck-reset_all *,
+	.ck-reset_all a,
```

Comment:
Is it necessary here to use `1-1` CSS specificity for `<a>` and `<textarea>` elements? 

As we talked it was an [issue with CK4](https://dev.ckeditor.com/ticket/13119) (IE10 maybe?) because UA override, but as I checked [Chrome](https://trac.webkit.org/browser/trunk/Source/WebCore/css/html.css) and Firefox and there is no override situation.

Reply:
Probably it's a legacy code because I copied it straight from v4.